src/test/java/com/kainos/ea/backend/controllers/UserControllerTest.java

```diff
@@ -8,116 +7,94 @@
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
-
 import java.util.List;
-
 import static org.junit.jupiter.api.Assertions.*;
 
 @ExtendWith(MockitoExtension.class)
 public class UserControllerTest {
 
     @Mock
     private UserService userService;
-
     @Mock
     private BCryptPasswordEncoder passwordEncoder;
 
     @Test
-    public void when_verifyUser_called_with_valid_user_expect_true() {
+    public void when_verifyUserCalledWithValidUser_expect_true() {
         User validUser = new User("mail@email.com", "strong_password");
-        List<User> users = List.of(new User("mail@email.com", "strong_password"));
+        List<User> users = List.of(validUser);
         Mockito.when(userService.validateUser(validUser)).thenReturn(true);
         Mockito.when(userService.getUsers()).thenReturn(users);
         Mockito.when(passwordEncoder.matches("strong_password", "strong_password")).thenReturn(true);
         UserController userController = new UserController(userService, passwordEncoder);
-
```

Comment:
I didn't mind the white space here separating your typical logical groups of setup/call/assert within your unit tests. However, if you are removing them, you need to be consistent and do it for all test files. See src/test/java/com/kainos/ea/backend/services/UserServiceTest.java

Reply:
I think it should be consistent now